src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OppgaveService.kt

```diff
@@ -97,7 +97,7 @@ class OppgaveService(
         }
 
     private fun finnMappeForFremleggsoppgave(enhetsnummer: String?, behandlingId: UUID): Long? {
-        if (enhetsnummer == "4489") {
+        if (enhetsnummer == "4489" || enhetsnummer == "4483") {
             val mappenavnProd = "41 Revurdering"
             val mappenavnDev = "41 - Revurdering"
```

Comment:
Ikke relevant for denne pull requesten, men ikke så fan av dev - mappenavn i prodkode.

Reply:
Hva tenker du på ?

Reply:
val mappenavnDev = "41 - Revurdering"